samples/nrf5340/multiprotocol_rpmsg/src/main.c

```diff
@@ -265,9 +265,11 @@ void nrf_802154_serialization_error(const nrf_802154_ser_err_data_t *err)
 {
 	(void)err;
 	__ASSERT(false, "802.15.4 serialization error");
+	k_panic();
```

Comment:
> A kernel error is a software triggered fatal error invoked by `k_panic()`. This should be used to indicate that the Zephyr kernel is in an unrecoverable state.

I don't think that serialization error should be treated as a kernel unrecoverable state. It is possible that other threads run just fine while serialization crashes.

Maybe `k_oops()` instead? Serialization seems closer to application logic than to kernel state

> A kernel oops is a software triggered fatal error invoked by `k_oops()`. This should be used to indicate an unrecoverable condition in application logic.

Reply:
Also from fatal_types.h:
```
        /** Moderate severity software error */
        K_ERR_KERNEL_OOPS,

        /** High severity software error */
        K_ERR_KERNEL_PANIC,
```

Reply:
Changed to `k_oops` as you suggest, as well as in upstream PR https://github.com/zephyrproject-rtos/zephyr/pull/58173